### DISCUSSION TITLE
[firebase_remote_config] support for web?

### DISCUSSION THREAD

**Discussion by nerder** [firebase/flutterfire#3440] [Feature request] [answered]
As in the title, it will be possible to have web support for `firebase_remote_config`?

**Comment by Ehesp** [comment, marked as answer]
It's planned, however the package requires a rework to a federated plugin setup and upgrading to the lastest APIs. First priority is native, then web.

**Comment by nerder** [comment]
That's a great news @Ehesp just wanted to have a bit of roadmap.

Thank you so much for your fast reply!
